lib/install/management/finder.go

```diff
@@ -232,23 +232,39 @@ func (d *Dispatcher) FetchAndMigrateVCHConfig(vm *vm.VirtualMachine) (*config.Vi
 	return d.decryptVCHConfig(vm, newMap)
 }
 
+// SearchVCHs searches for VCHs in one of the following areas:
+//
+// computePath:  if a compute resource is provided then the search will be limited
+// to the resource pools of the compute resources found at the path
+//
+// datacenter:  if no compute resource is provided then search across all compute
+// resources in the sessions datacenter.  This requires the user to specify the datacenter
+// via the target flag
+//
+// all datacenters:  if the other options aren't available then search across every compute
+// resource in each vSphere datacenter
 func (d *Dispatcher) SearchVCHs(computePath string) ([]*vm.VirtualMachine, error) {
 	defer trace.End(trace.Begin(computePath, d.op))
 	if computePath != "" {
-		return d.searchVCHsFromComputePath(computePath)
+		d.op.Infof("Searching for VCHs in computePath %s", computePath)
+		return d.search(computePath)
```

Comment:
What do we do if the compute path isn't nil but the datacenter isn't set? Do we filter for that compute path in all datacenters? If it's scoped to a datacenter then we could print which datacenter we're listing for so it's explicit.

Also, without `...` on the end will this still find VCHs that are several levels nested from the specified compute path?